src/css/_about.scss

```diff
@@ -0,0 +1,63 @@
+.logo {
+	height: 60px;
+	width: auto;
+	padding: 10px;
+}
+
+.topnav {
+	background-color: #412155;
+	overflow: hidden;
+}
+
+.topnav a {
+	float: right;
+	display: block;
+	color: #f2f2f2;
+	text-align: center;
+	padding: 20px 20px;
+	margin: 10px;
+	text-decoration: none;
+	font-family: 'Sofia Pro Soft', sans-serif;
+	font-size: 20px;
+}
+
+.topnav a:hover {
+	color: purple;
+}
+
+.topnav a.active {
+	color: white;
+}
+
+.topnav .icon {
+	display: none;
+}
+
+@media screen and (max-width: 600px) {
+	.topnav a:not(:first-child) {
+		display: none;
+	}
```

Comment:
This is a neat way to write hiding the navigation at mobile sizes, I haven't seen this pattern before! Great use of the `not` selector. I think mobile navigation styles are a good case for a max-width selector, just be judicious and try to keep most of your styles mobile first.